README.md

```diff
@@ -1,44 +1,38 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project
-> About this project.
+![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white) ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white) ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
 
+# Microverse Module 1 Capstone Project ⚙️
+> This is a non commercial project. For learning purpose based on an original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07) under creative commons license.
+In this project we were required to follow design guidelines while coding a semantic HTML structure, layout and colors in CSS and dynamic section (speakers) and functions (mobile menu) by Javascript following industry best practices.
 
-## Built With
 
+## Tech ⌘⇧
 - **HTML & CSS best practices:** Correct use of tags, elements, properties and syntax.
 - **GitHub flow:**  Correct use of Branches for deployment and features development.
-- **Linters Check:** Local and Pull Request check for errors, bugs and stylistic errors in code.
+- **Linters Check:** Local and Pull Request Github workflows to check for stylistic errors, bugs and syntax errors in code.
 
 ## Live Demo 
+[Github Pages Demo](https://raminka13.github.io/Capstone-M1/)
 
-[Live Demo Link](https://raminka13.github.io/M1-Portfolio/)
-
```

Comment:
- [ ] Kindly add the presenting video link to the README file and as the previous code reviewer said that you should talk about the project code and especially highlight a piece of the code you find interesting or are proud of making.